db/schema.rb

```diff
@@ -82,9 +82,9 @@
     t.string "national_insurance_number", limit: 9
     t.string "email_address", limit: 256
     t.boolean "mostly_teaching_eligible_subjects"
+    t.datetime "submitted_at"
     t.string "bank_sort_code", limit: 6
     t.string "bank_account_number", limit: 8
-    t.datetime "submitted_at"
```

Comment:
We should avoid these unexpected column order changes in `schema.rb`, otherwise they will dance back and forth as others run migrations. They happen when locally migrations have been run out of order (normally when switching between branches and running migrations before they've been merged into master). Something to keep an eye out in the future. I'll push up a PR to fix this.